stories/Atom/Reach-element/Table/table.jsx

```diff
@@ -1,35 +1,53 @@
 import React from 'react';
 import './table.scss';
 
-export const TableTag = ({ text , tdtext, details}) => {
-  return (
-   <table>
-      <tr>
-         <th>{text}</th>
-         <th>{text}</th>
-         <th>{text}</th>
-      </tr>
-      <tr>
-         <td>{tdtext}	</td>
-         <td>{tdtext}</td>
-         <td>{details}</td>
-      </tr>
-       <tr>
-         <td>{tdtext}	</td>
-         <td>{tdtext}</td>
-         <td>{details}</td>
-      </tr>
-       <tr>
-         <td>{tdtext}	</td>
-         <td>{tdtext}</td>
-         <td>{details}</td>
-      </tr>
-       <tr>
-         <td>{tdtext}	</td>
-         <td>{tdtext}</td>
-         <td>{details}</td>
-      </tr>
+const TableScroll = ({ link, children }) => (link == 'scroll' ?
```

Comment:
@harishkumar-srijan can you please change link variable name to something like tableType

Reply:
Done